### PR TITLE
feat: Feature Flag for ENS

### PR DESCRIPTION
**Pull request by malaniz**
Feature Flag for ENS

If REACT_APP_FF_ENS exist then will be turned on, if not defined will be turned OFF.

### CODE REVIEW

**Review by cazala — CHANGES_REQUESTED** [review]
i think we should also add this env var to the `now.json` file. Also maybe there are other missing env vars from the ens branch that we should add there so the feature works on the vercel previews

File: src/components/LandDetailPage/LandDetailPage.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Profile from 'components/Profile'
 import Scene from './Scene'
 import { Props, State } from './LandDetailPage.types'
 import './LandDetailPage.css'
+import { env } from 'decentraland-commons'
```
Comment (cazala) [inline src/components/LandDetailPage/LandDetailPage.tsx:14]: nip tick: cam we move this import to the top? we usually have the modules on top and the relative imports below

**Review by cazala — APPROVED** [review]

File: src/components/LandDetailPage/LandDetailPage.tsx [hunk 2]
```diff
@@ -146,7 +149,12 @@ export default class LandDetailPage extends React.PureComponent<Props, State> {
                             text={t('land_detail_page.set_operator')}
                             onClick={() => onNavigate(locations.landOperator(land.id))}
                           />
-                          <Dropdown.Item text={t('land_detail_page.set_link')} onClick={() => onNavigate(locations.landEns(land.id))} />
+                          {FF_ENS ? (
```
Comment (nicosantangelo) [inline src/components/LandDetailPage/LandDetailPage.tsx:152]: Sorry I saw this one second, but could we change it to look the same as the one juanca did? here https://github.com/decentraland/builder/pull/1151/files#diff-e6f76a45490df7734399812bad8e19dda9c880883a47f3101bf25025124d65afR20

it's super minor, but for consistency sake

File: src/routing/Routes.tsx [hunk 3]
```diff
@@ -79,7 +81,7 @@ export default class Routes extends React.Component<Props, State> {
             <Route exact path={locations.land()} component={LandPage} />
             <Route exact path={locations.landDetail()} component={LandDetailPage} />
             <Route exact path={locations.landTransfer()} component={LandTransferPage} />
-            <Route exact path={locations.landEns()} component={LandEnsPage} />
+            { FF_ENS && <Route exact path={locations.landEns()} component={LandEnsPage} /> }
```
Comment (nicosantangelo) [inline src/routing/Routes.tsx:84]: Same comment as above

**Review by nicosantangelo — APPROVED** [review]
LGTM super minor comment. I'll approve anyways

**Review by nicosantangelo — APPROVED** [review]
🔥

**Pull request merged**